lib/handlers/commitHandlerFactory.js

```diff
@@ -0,0 +1,43 @@
+const {
+  abci: {
+    ResponseCommit,
+  },
+} = require('abci/types');
+const {
+  CommitTransactionRequest,
+} = require('@dashevo/drive-grpc');
+
+/**
+ * commit ABCI handler
+ * @param {AppState} state
+ * @param {UpdateStatePromiseClient} updateStateClient
+ * @returns {commitHandler}
+ */
+module.exports = function commitHandlerFactory(state, updateStateClient) {
+  /**
+   * @typedef commitHandler
+   * @returns {Promise<abci.ResponseCommit>}
+   */
+  async function commitHandler() {
```

Comment:
I feel like we receive block height and block hash in the request object. Why we get it from the state?